src/main/java/org/assertj/core/api/Assert.java

```diff
@@ -272,6 +272,32 @@
    */
   SELF usingDefaultComparator();
 
+  /**
+   * Uses an {@link InstanceOfAssertFactory} to verify that the actual value is an instance of a given type and to produce
+   * a new {@link Assert} narrowed to that type.
+   * <p>
+   * {@link InstanceOfAssertFactories} provides static factories for all the types supported by {@link Assertions#assertThat}.
+   * Additional factories can be created with custom {@code InstanceOfAssertFactory} instances.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertions will pass
+   * assertThat((Object) &quot;abc&quot;).instanceOf(InstanceOfAssertFactories.STRING).startsWith(&quot;ab&quot;);
```

Comment:
let's extract the actual value as an `Object` local variable for readability and to emphasize that are able to chain specific type assertions from an object